packages/components/src/focusable-iframe/index.js

```diff
@@ -23,21 +22,15 @@ export default function FocusableIframe( { iframeRef, onFocus, ...props } ) {
 				return;
 			}
 
-			const focusEvent = new FocusEvent( 'focus', { bubbles: true } );
-
-			iframe.dispatchEvent( focusEvent );
-
-			if ( onFocus ) {
-				onFocus( focusEvent );
```

Comment:
We no longer need this since React will fire the right handlers when the element receives focus.

Reply:
Note that React will even call the right handlers with `new FocusEvent`. I'm not sure why this was ever added, but it was assumed React wouldn't pick it up.